integration-cli/docker_cli_run_unix_test.go

```diff
@@ -1155,6 +1155,15 @@ func (s *DockerSuite) TestRunNoNewPrivSetuid(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunAmbientCapabilities(c *check.C) {
+	testRequires(c, DaemonIsLinux, ambientCapabilities)
+
+	// test that a non root user can gain capabilities
+	runCmd := exec.Command(dockerBinary, "run", "--user", "1000", "--cap-add chown", "busybox", "chown", "100", "/tmp")
```

Comment:
Should we have a failing test as well, to verify that it won't work without this?